### PR TITLE
docs(core): fix anchor links in documentation for Core components

### LINKED ISSUE #5960: bug (core): do not work anchors for examples in few components
#### Briefly describe your proposal.
Do not work anchor links for example in next components:

- Busy indicator
- Facets
- Info label
- Input Group (only first anchor)
- List with Bylin (links redirect to the List component)
- Message Page
- Message Strip
- Object Identifier
- Object Marker
- Object Number
- Object Status
- Pagination (wrong path to the examples)
- Status Indicator
- Time (wrong path to In Internationalization example)
- TimePicker (anchors direct to Time component)
- Title (anchors direct to Tile component)
- Dinamyc Page
- Dinamyc Side Content
- Flexible Card Layout
- Flexible Column Layout
- Layout Grid

#### If this is a bug, please provide steps for reproducing it.
1. Open a component page
2. Hover on any example name and on the anchor
3. Click on the anchor
Expected result: link is correct, after clicking on achor - you are directed to the right page & right example
Actual result: nothing happened, link does not work
#### Is there anything else we should know?
Also not working for some links in:
- Theme Switcher
- Global Config
- Table
- Side Navigation
- Rating indicator
- Icon
- Combobox

### PR DESCRIPTION
**Pull request by kavya-b**
#### Please provide a link to the associated issue.
Closes #5960 

#### Please provide a brief summary of this pull request.
This PR fixes incorrect anchor links for the documentation examples of Core components. All pages in Core(even other than the ones captured in the issue) have been verified and fixed where issue found.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [n/a] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [n/a] update `README.md`
- [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/busy-indicator/busy-indicator-docs.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<fd-docs-section-title id="default" componentName="BusyIndicator">
+<fd-docs-section-title id="default" componentName="busyIndicator">
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/busy-indicator/busy-indicator-docs.component.html:1]: not consistent with the others which are using kebab case. For example: `fixed-card-layout`.

**Comment by kavya-b** [comment]
have also fixed ids to have kebab-case and removed unnecessary `"''"` binding from `id` and `componentName` in most places

File: apps/docs/src/app/core/component-docs/tile/tile-docs.component.html [hunk 2]
```diff
@@ -105,7 +105,7 @@
 </component-example>
 <code-example [exampleFiles]="slide"></code-example-->
 
-<!-- TODO: wip fd-docs-section-title [id]="'line'" [componentName]="'tile'">
+<!-- TODO: wip fd-docs-section-title id]="'line'" [componentName="tile">
```
Comment (olshansky) [inline apps/docs/src/app/core/component-docs/tile/tile-docs.component.html:108]:
```suggestion
<!-- TODO: wip fd-docs-section-title id="line" [componentName="tile">
```

File: apps/docs/src/app/core/component-docs/tile/tile-docs.component.html [hunk 3]
```diff
@@ -95,7 +95,7 @@
 </component-example>
 <code-example [exampleFiles]="feed"></code-example>
 
-<!--TODO: wip fd-docs-section-title [id]="'slide'" [componentName]="'tile'">
+<!--TODO: wip fd-docs-section-title id]="'slide'" [componentName="tile">
```
Comment (olshansky) [inline apps/docs/src/app/core/component-docs/tile/tile-docs.component.html:98]:
```suggestion
<!--TODO: wip fd-docs-section-title id="slide" [componentName="tile">
```

**Review by olshansky — APPROVED** [review]

**Review by InnaAtanasova — APPROVED** [review]

File: apps/docs/src/app/core/component-docs/tile/tile-docs.component.html (hunk 2 above)
Comment (kavya-b) [reply apps/docs/src/app/core/component-docs/tile/tile-docs.component.html:108]: thanks for catching this! Fixed it.

File: apps/docs/src/app/core/component-docs/tile/tile-docs.component.html (hunk 3 above)
Comment (kavya-b) [reply apps/docs/src/app/core/component-docs/tile/tile-docs.component.html]: fixed

**Comment by puru-hk** [comment]
Looks Fine

**Review by puru-hk — APPROVED** [review]

**Pull request merged**